subprojects/core-api/src/main/java/org/gradle/api/file/DuplicatesStrategy.java

```diff
@@ -53,7 +53,13 @@
      * <p> 
      * Use this strategy when duplicates are an error condition that should cause the build to fail.
      */
-    FAIL
-
+    FAIL,
 
+    /**
+     * The default strategy, which is inherited from the parent copy spec, if any,
+     * or {@link DuplicatesStrategy#INCLUDE} if the copy spec has no parent.
```

Comment:
We know when we create the root spec, so we could just as well set it to `INCLUDE` already, no?

Reply:
The problem is detecting when to use the parent resolver strategy here:

https://github.com/gradle/gradle/blob/d2183df58d794f7016c67f23801d6045527534f3/subprojects/core/src/main/java/org/gradle/api/internal/file/copy/DefaultCopySpec.java#L620

I think the current solution is ok.